ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/type/TypeMapperImpl.java

```diff
@@ -46,14 +61,97 @@
  */
 public class TypeMapperImpl implements TypeMapper {
 
+    public static final String COMPONENTS_SCHEMAS = "#/components/schemas/";
+    public static final String DEFINITIONS = "#/definitions/";
+    public static final String DEFINITIONS_FIELD = "definitions";
+    public static final String JSON_SCHEMA_FIELD = "$schema";
+    public static final String JSON_SCHEMA_VERSION = "http://json-schema.org/draft-04/schema#";
```

Comment:
Is there compatibility for OAS v3.1 with the JSON Schema draft version? The previous mention was for OAS 3.0 with JSON Draft-05 support and the Draft 4 meta-schemas.

Reply:
I could not found an information regarding this. This is added by considering the minimal support. Also we only adding specific fields from our side, so it should be compatible with the latest version as well. (Btw the actual schema is not an optimised Json Schema, as mentioned in the original issue, the JSON specific schema generation should be supported from jsondata package. And here we only provide an API as a workaround)

Reply:
Yeah, This is fine because we're supporting the initial stage of this effort as a workaround, and +1  to the capability come from the `jsondata` package.